osu.Game.Rulesets.Osu/Objects/Slider.cs

```diff
@@ -130,6 +131,17 @@ private void createTicks()
                     var distanceProgress = d / length;
                     var timeProgress = reversed ? 1 - distanceProgress : distanceProgress;
 
+                    var firstSample = Samples.FirstOrDefault();

```

Comment:
How about `Samples.FirstOrDefault(s => s.Name == SampleInfo.HIT_NORMAL)`. I believe this is more correct in terms of what stable does?

Reply:
you'd really hope the first sample in the list is hitnormal, but i have no issue with adding this.

Reply:
What's the reasoning for hitnormal being first?

Reply:
because it's the main sample for the hitobject?

Reply:
actually i may not change this. if a hitobject ends up not having a hitnormal it will make all ticks silent, which is probably not expected behaviour.

Reply:
`Samples.FirstOrDefault(...) ?? Samples.FirstOrDefault()`, i.e prioritise the hitnormal?

Reply:
honestly i'd rather just have some stable sort so we don't have to do this everywhere it's expected behaviour

Reply:
It's not "expected behaviour" by anything else other than osu!... All other hitobjects dont care which sample is first or not.